src/java.management/share/classes/java/lang/management/MemoryMXBean.java

```diff
@@ -267,6 +267,24 @@ public interface MemoryMXBean extends PlatformManagedObject {
      */
     public MemoryUsage getNonHeapMemoryUsage();
 
+    /**
+     * Returns the CPU time used by all garbage collection threads.
+     *
+     * <p> This include time since genesis, so the value can be
```

Comment:
The “so the” is not too obvious to me. Maybe simplify it a bit “This includes time since genesis and counts activities even before the first collection cycle.”?

Reply:
Thanks @ecki for pointing that out :) I make a simplification whenever the CSR discussion (https://bugs.openjdk.org/browse/JDK-8368529) have reached a consensus where this method should live.